packages/core/src/llms/live/message-handler.ts

```diff
@@ -0,0 +1,80 @@
+import type {
+  MessageContentAudioDetail,
+  MessageContentDetail,
+  MessageContentImageDataDetail,
+  MessageContentVideoDetail,
+} from "../type";
+import type { MessageSender } from "./sender";
+
+export abstract class MessageHandler {
```

Comment:
not needed if providers just implement `MessageSender` and live.ts directly uses it